File: src/integrationTest/java/info/idgst/selenium/MainPageIntegrationTest.java
package info.idgst.selenium;

import org.junit.Test;

import static org.hamcrest.MatcherAssert.assertThat;
import static org.hamcrest.Matchers.is;

/**
 * It's an example of Selenium Tests for Main page.
 * <p>
 * // TODO should be updated.
 *
 * @author Aliaksei Bahdanau.
 */
public class MainPageIntegrationTest extends AbstractFluentIntegrationTest {

    @Test
    public void hasPageTitle() {
        goTo("/");
        assertThat(findFirst("h5").getText(), is("Digests archive"));
    }
}

File: src/integrationTest/kotlin/info/devbug/selenium/MainPageIntegrationTest.kt
package info.devbug.selenium

import org.junit.Test

/**
 * Selenium test for main page.
 *
 * @author Aliaksei Bahdanau.
 */
class MainPageIntegrationTest : AbstractFluentIntegrationTest() {

    @Test
    fun hasPageTitle() {
        goTo("/")
        assert(findFirst("h5").text == "Digests archive")
    }
}